### PR TITLE
Volume creation in parent test and param handler changes

### PR DESCRIPTION
**Pull request by srijan-sivakumar**
Multiple code changes:
1. Volume creation method modified to handle replica,
distributed and distributed-replica volume with servers
having single brick root.
2. Param handler methods converted from class methods to
regular methods so that gc doesn't do a cleanup after
object goes out of scope.

Fixes: #209
Updates: #186

Signed-off-by: srijan-sivakumar <ssivakum@redhat.com>

### CODE REVIEW

File: common/ops/gluster_ops/volume_ops.py [hunk 1]
```diff
@@ -68,30 +68,19 @@ def volume_unmount(self, path: str, node: str = None):
...
-                    - arbiter_count : (int)|None
-                    - stripe_count : (int)|None
-                    - disperse_count : (int)|None
-                    - disperse_data_count : (int)|None
-                    - redundancy_count : (int)|None
-                    - transport_type : tcp|rdma|tcp,rdma|None
-                    - ...
+            will get executed with force option.
+
         Returns:
```
Comment (aujjwal-redhat) [inline common/ops/gluster_ops/volume_ops.py:84]: NIT: returns 0 :)

**Review by aujjwal-redhat — APPROVED** [review]
LGTM

File: common/ops/gluster_ops/volume_ops.py (hunk 1 above)
Comment (srijan-sivakumar) [reply common/ops/gluster_ops/volume_ops.py:84]: actually it should have been ret...

**Review by nishith-vihar — APPROVED** [review]
LGTM

**Pull request merged**